components/03-modules/catalog-grid-item/_catalog-grid-item.scss

```diff
@@ -68,19 +78,21 @@ $catalog-gird-item__primary-form-margin     : 0 !default;
 
     &__image-wrapper {
         background: #f9f9f9;
```

Comment:
i know it's legacy code but please move to var

Reply:
Moved